openapi/azure.openai.deployment/Module.md

```diff
@@ -0,0 +1,49 @@
+## Overview
+This is a generated connector from [Azure OpenAI Deployments API](https://learn.microsoft.com/en-us/rest/api/cognitiveservices/azureopenaistable/deployments/) OpenAPI specification.
+
+The Azure OpenAI Deployments REST API endpoints will deploy OpenAI models in Azure OpenAI resource. 
+
+## Prerequisites
+- Create an [Azure](https://azure.microsoft.com/en-us/features/azure-portal/) account.
+- Create an [Azure OpenAI resource](https://learn.microsoft.com/en-us/azure/cognitive-services/openai/how-to/create-resource).
+- Obtain the tokens. Refer [this](https://learn.microsoft.com/en-us/azure/cognitive-services/openai/reference#authentication) guide to learn how to generate and use tokens.
```

Comment:
```suggestion
- Obtain the tokens. Refer to [this](https://learn.microsoft.com/en-us/azure/cognitive-services/openai/reference#authentication) guide to learn how to generate and use tokens.
```

Reply:
Modified the PR according to the comment